crates/jstzd/src/task/jstzd.rs

```diff
@@ -182,6 +186,7 @@ struct ServerState {
     jstzd_config_json: serde_json::Map<String, serde_json::Value>,
     jstzd: Option<Jstzd>,
     server_handle: Option<JoinHandle<()>>,
+    tx: Option<mpsc::Sender<()>>,
```

Comment:
we only need to send a single shutdown signal so we can use `oneshot`

Reply:
nit:
shutdown_tx 
to make it more explicit

Reply:
what's the problem with using mpsc::Sender?

Reply:
oneshot is more efficient as it's optimized for single-use :) i assume for shutdown signal we just need to send one message, once.